products/sle/main.pm

```diff
@@ -993,7 +993,6 @@ sub load_online_migration_tests() {
     # stop packagekit service and more
     loadtest "online_migration/sle12_online_migration/online_migration_setup";
     loadtest "online_migration/sle12_online_migration/register_system";
-    loadtest "online_migration/sle12_online_migration/repos_check";
```

Comment:
This test module does not seem to used anywhere now, delete? Also I think this is an unrelated change.